lib/index.js

```diff
@@ -39,5 +39,36 @@ function purify(htmlInput, options, cb) {
   });
 }
 
+/**
+ * Strips provided prefix and postfix from id and class attributes
+ *
+ * @param String htmlInput
+ * @param Object options
+ * @cb err, string
+ */
+function revertPurify(htmlInput, options, cb) {
```

Comment:
I didn't see any unit tests for this function, can you add one to ensure this function works? While the other tests you have ensure smaller parts of it function.